definitions/region/connections.yaml

```
@@ -0,0 +1,192 @@
+- North Sea|Borssele>North Sea|Hollandsee Kust
+- North Sea|Borssele>North Sea|Nordsoen
+- North Sea|Borssele>North Sea|Sorlige Nordsjo I
+- North Sea|Borssele>North Sea|Sorlige Nordsjo II
+- North Sea|Borssele>North Sea|Utsira Nord
+- Bosnia and Herzegovina>Croatia
+- Bosnia and Herzegovina>Italy
+- Bosnia and Herzegovina>Serbia
+- Bulgaria>Greece
+- Bulgaria>Macedonia
+- Bulgaria>Romania
+- Bulgaria>Serbia
+- Croatia>Hungary
+- Croatia>Italy
+- Croatia>Serbia
+- Croatia>Slovenia
+- Czechia>Germany
+- Czechia>Poland
+- Czechia>Slovakia
+- Denmark>United Kingdom
+- Denmark>Netherlands
+- Denmark>Norway|Sorland
+- Denmark>North Sea|Nordsoen
+- Denmark>Poland
+- Denmark>Sweden
+- North Sea|Dogger Bank>North Sea|Borssele
+- North Sea|Dogger Bank>North Sea|East Anglia
+- North Sea|Dogger Bank>North Sea|Helgolander Bucht
+- North Sea|Dogger Bank>North Sea|Hollandsee Kust
+- North Sea|Dogger Bank>North Sea|Hornsea
+- North Sea|Dogger Bank>North Sea|Nordsoen
+- North Sea|Dogger Bank>North Sea|Norfolk
+- North Sea|Dogger Bank>North Sea|Outer Dowsing
+- North Sea|Dogger Bank>North Sea|Sorlige Nordsjo I
+- North Sea|Dogger Bank>North Sea|Sorlige Nordsjo II
+- North Sea|Dogger Bank>North Sea|Utsira Nord
+- North Sea|East Anglia>North Sea|Borssele
+- North Sea|East Anglia>North Sea|Helgolander Bucht
+- North Sea|East Anglia>North Sea|Hollandsee Kust
+- North Sea|East Anglia>North Sea|Nordsoen
+- North Sea|East Anglia>North Sea|Sorlige Nordsjo I
+- North Sea|East Anglia>North Sea|Sorlige Nordsjo II
+- North Sea|East Anglia>North Sea|Utsira Nord
+- Estonia>Finland
+- Estonia>Latvia
+- Estonia>Sweden
+- Finland>Norway|Troms
+- Finland>Sweden
+- North Sea|Firth of Forth>North Sea|Borssele
+- North Sea|Firth of Forth>North Sea|Dogger Bank
+- North Sea|Firth of Forth>North Sea|East Anglia
+- North Sea|Firth of Forth>North Sea|Helgolander Bucht
+- North Sea|Firth of Forth>North Sea|Hollandsee Kust
+- North Sea|Firth of Forth>North Sea|Hornsea
+- North Sea|Firth of Forth>North Sea|Nordsoen
+- North Sea|Firth of Forth>North Sea|Norfolk
+- North Sea|Firth of Forth>North Sea|Outer Dowsing
+- North Sea|Firth of Forth>North Sea|Sorlige Nordsjo I
+- North Sea|Firth of Forth>North Sea|Sorlige Nordsjo II
+- North Sea|Firth of Forth>North Sea|Utsira Nord
+- France>United Kingdom
+- France>Ireland
+- France>Italy
+- France>Luxembourg
+- Germany>Denmark
+- Germany>France
+- Germany>North Sea|Helgolander Bucht
+- Germany>Luxembourg
+- Germany>Netherlands
+- Germany>Norway|Sorland
+- Germany>Poland
+- Germany>Sweden
+- United Kingdom>North Sea|Dogger Bank
+- United Kingdom>North Sea|East Anglia
+- United Kingdom>North Sea|Firth of Forth
+- United Kingdom>North Sea|Hornsea
+- United Kingdom>Ireland
+- United Kingdom>North Sea|Moray Firth
+- United Kingdom>Netherlands
+- United Kingdom>Norway|Sorland
+- United Kingdom>Norway|Vestmidt
+- United Kingdom>North Sea|Norfolk
+- United Kingdom>North Sea|Outer Dowsing
+- Greece>Italy
+- Greece>Macedonia
```

The region is called "North Macedonia", please fix everywhere.
```suggestion
- Greece>North Macedonia
```